<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="subName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solve for</string>
    </property>
   </widget>
   <widget class="QLabel" name="indexChange">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidgetArea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="symbol">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>27</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lengthValue"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Result">
         <property name="font">
          <font>
           <pointsize>27</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelVal1">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>L      length</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="LabelWorkings">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelVal2">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>W    width</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="widthValue"/>
       </item>
       <item row="1" column="2" rowspan="3">
        <widget class="QGraphicsView" name="area_diagram"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="LabelSolution">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="CustomLineEdit" name="input2_2">
         <property name="placeholderText">
          <string>Enter value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Eqaution_symbol_2">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>27</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>P =</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="areaResult_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="areaLabelLength_2">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>L      length</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CustomLineEdit" name="input1_2">
         <property name="placeholderText">
          <string>Enter value</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="areaLabelWidth_2">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>W    width</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="areaLabelWorkings_3">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="displayShape_2">
         <property name="text">
          <string>display shape</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="areaLabelSolution_2">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Solution</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QGraphicsView" name="area_diagram_2"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>471</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="CustomLineEdit" name="input1_3">
         <property name="placeholderText">
          <string>Enter value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QGraphicsView" name="area_diagram_3"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="areaResult_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CustomLineEdit" name="input2_3">
         <property name="placeholderText">
          <string>Enter value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="areaLabelLength_3">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>L      length</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="displayShape_3">
         <property name="text">
          <string>display shape</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="areaLabelWidth_3">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>W    width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Eqaution_symbol_3">
         <property name="baseSize">
          <size>
           <width>3</width>
           <height>3</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>27</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>L =</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="areaLabelSolution_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Using this formula</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="areaLabelSolution_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>P=2( l + w )</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="areaLabelSolution_3">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Solution</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="areaLabelWorkings_4">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Select Function</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Area</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perimeter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Length</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="calculatorName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
